api/mapping/templates/mapping/upload_scan_report.html

```diff
@@ -6,70 +6,37 @@
         <li class="breadcrumb-item"><a href="{% url 'scan-report-list' %}">Scan
-            reports</a></li>
+                reports</a></li>
     </ol>
 </nav>
-<h2>New Scan Report</h2>
 
-<form method="post" name="simpleForm" id="simpleForm" action="#" enctype="multipart/form-data">{% csrf_token %}
-
-    <div class="mb-3">
-        {{ form.parent_dataset.errors }}
-        <label class="form-label" id='parent_dataset'>{{ form.parent_dataset.label }}</label>
-        {{ form.parent_dataset }}
-    </div>
-    
-    <div class="mb-3">
-        {{ form.dataset.errors }}
-        <label class="form-label">{{ form.dataset.label}}</label>
-        {{ form.dataset }}
-    </div>
-
-    <div class="mb-3">
-
-        <label class="form-label">{{ form.scan_report_file.label }}</label>
-        {{ form.scan_report_file }}
-        {%if form.scan_report_file.errors%}
-        <div class="alert alert-danger" role="alert">{{ form.scan_report_file.errors }}</div>
-        {%endif%}
-
-    </div>
-
-    <div class="mb-3">
-
-        <label class="form-label">{{ form.data_dictionary_file.label }}</label>
-        {{ form.data_dictionary_file }}
-        {%if form.data_dictionary_file.errors%}
-        <div class="alert alert-danger" role="alert">{{ form.data_dictionary_file.errors }}</div>
-        {%endif%}
-
-    </div>
-    <div class="row mx-auto">
-        <div class="spinner-border mx-auto d-none" name="spinner" id="spinner" role="status">
-          <span class="sr-only">Uploading...</span>
-        </div>
-    </div>
-    <button type="submit" class="btn btn-primary" id="btn">Submit</button>
-
-</form>
-<script>
-
-$("#simpleForm").on('submit',function(ev) {      
-        $.ajax({
-        type: "POST",
-        data: {
-        'csrfmiddlewaretoken': '{{ csrf_token }}',
-        },
-        beforeSend: function (data) {
-        $('#spinner').addClass('d-none');
-        },
-        success: function (data) {
-            $('#spinner').removeClass('d-none');
-        },
-        error: function(data,XMLHttpRequest, textStatus, errorThrown){
-            $('#spinner').addClass('d-none');
+<div>
+    <div id='root'></div>
+    <script type="text/javascript">
+        var a = "{{a}}"
+        var u = "{{u}}"
+
+        var uploadScanReport = async function (formData) {
+            formData.append('csrfmiddlewaretoken', '{{ csrf_token }}')
+            try {
+                const res = await $.ajax({
+                    type: "POST",
+                    processData: false,
+                    contentType: false,
+                    data: formData,
+                    enctype: 'multipart/form-data',
+                });
+                if (res.status == "form-invalid") {
+                    throw {statusText:"There was a form error",...res}
+                }
+                return res
+            }
+            catch (err) {
+                throw err
+            }
```

Comment:
In #404 I made it so that the CSRF token can be obtained from the cookies. This then means the `usePost` and `usePatch` function in `values.js` automatically get the CSRF token for you, and you should be able to send the form data using `usePost` no problem. 
We should merge #404's code into here and ditch this ajax call here in favour of `usePost`